solr/solr-ref-guide/src/stream-api.adoc

```diff
@@ -0,0 +1,210 @@
+= Stream Request Handler API
+:page-toclevels: 1
+:page-tocclass: right
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+
+These API commands work with the `/stream` request handler.
+
```

Comment:
Since there are only really 2 types of actions possible - 1 to list all expressions available, and 4 to manipulate daemon streams - I think it might be helpful for users to state that somewhat limited scope upfront.

Also a link from the daemon expression to this new page would be appropriate IMO, and less importantly a link to the Stream handler documentation (unless this is intended to be a child of that page - not clear where this is intended to live).

Reply:
Also, it seems that the v2 API structure is not supported with this API? If that's true, we might want to state that somewhere outright.

Reply:
I reworked the intro, and used the `NOTE` to call out the lack of following v2 API structure.   Also, there is a link from the Daemon expression detail to this page and vice versa, and I modified the `streaming-expressions.adoc` to have `stream-api.adoc` be a child page.